kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -733,7 +734,39 @@ struct RemoveMetadata {
     if (!status.isOK()) {
       throw std::runtime_error{"Failed to delete metadata and state transfer data: " + status.toString()};
     }
-    return toJson(std::string{"result"}, std::string{"true"});
+    // Once we managed to remove the metadata, we must start a new epoch (which means to add an epoch block)
+    uint64_t epoch{0};
+    {
+      auto value =
+          adapter.getLatest(kvbc::kConcordInternalCategoryId, std::string{kvbc::keyTypes::reconfiguration_epoch_key});
+      if (value) {
+        const auto &data = std::get<categorization::VersionedValue>(*value).data;
+        ConcordAssertEQ(data.size(), sizeof(uint64_t));
+        epoch = concordUtils::fromBigEndianBuffer<uint64_t>(data.data());
+      }
+    }
+    uint64_t last_executed_sn{0};
+    {
+      auto value = adapter.getLatest(kvbc::kConcordInternalCategoryId, std::string{kvbc::keyTypes::bft_seq_num_key});
+      if (value) {
+        const auto &data = std::get<categorization::VersionedValue>(*value).data;
+        ConcordAssertEQ(data.size(), sizeof(uint64_t));
+        last_executed_sn = concordUtils::fromBigEndianBuffer<uint64_t>(data.data());
+      }
+    }
+    epoch += 1;
+    std::string epoch_str = concordUtils::toBigEndianStringBuffer(epoch);
+    concord::kvbc::categorization::VersionedUpdates ver_updates;
+    ver_updates.addUpdate(std::string{kvbc::keyTypes::reconfiguration_epoch_key}, std::move(epoch_str));
+    std::string sn_str = concordUtils::toBigEndianStringBuffer(last_executed_sn);
```

Comment:
Do we need to add the last executed sequence number with the same value again in the new block?